handler.go

```diff
@@ -225,9 +228,16 @@ func (h *Handler) buildUpstreamRequest(req *http.Request) (*http.Request, error)
 		return nil, err
 	}
 
+	// WORKAROUND S3CMD which dont use white space before the some commas in the authorization header
+	fakeAuthorizationStr := fakeReq.Header.Get("Authorization")
+	// Sanitize fakeReq to remove white spaces before the comma signature
+	authorizationStr := strings.Replace(req.Header["Authorization"][0], ",Signature", ", Signature", 1)
+	// Sanitize fakeReq to remove white spaces before the comma signheaders
```

Comment:
is this comment in the correct direction?
the `strings.Replace` function call is "adding" a space if it is missing. Or am I reading this wrong?